src/test/java/SoloTech/StockFlow/order/OrderServiceTest.java

```diff
@@ -138,122 +139,124 @@ void createOrderTest() {
-        when(localCache.getIfPresent(cacheKey)).thenReturn(null);
 
-        // When
-        Order result = orderService.updateOrder(orderId, dto);
+        // when
+        Order result = orderService.updateOrder(orderId, updateDto);
 
-        // Then
-        assertEquals(updatedOrder, result);
-        verify(localCache).put(cacheKey, updatedOrder);
-        verify(cachePublisher).publish("cache-sync", message);
+        // then
+        assertNotNull(result);
+        assertEquals(5L, result.getQuantity());
+        verify(orderRepository).findByOrderId(orderId);
+        verify(mapper).updateValue(existingOrder, updateDto);
         verify(orderRepository).save(existingOrder);
-        verify(mapper).updateValue(existingOrder, dto);
+    }
+
+    @Test
+    void updateOrder_NoOrder() {
+        // given
+        String orderId = "NOT_FOUND";
+        OrderDto dto = OrderDto.builder().quantity(3L).build();
+
+        when(orderRepository.findByOrderId(orderId)).thenReturn(Optional.empty());
+
+        // when & then
+        EntityNotFoundException exception = assertThrows(EntityNotFoundException.class, () ->
+                orderService.updateOrder(orderId, dto));
+
+        assertTrue(exception.getMessage().contains("Order not found"));
+        verify(orderRepository).findByOrderId(orderId);
     }
 
     @Test
     void deleteOrderTest() {
-        // Given
-        String orderId = "1234567890";
-        String cacheKey = ORDER_KEY_PREFIX + orderId;
-        Order mockOrder = Order.builder()
+        // given
+        String orderId = "ORD2001";
+        Order existingOrder = Order.builder()
                 .id(1L)
                 .orderId(orderId)
-                .storeId("S12345")
-                .productId("P56789")
-                .stockId("STK98765")
-                .quantity(2L)
                 .build();
 
-        when(orderRepository.findByOrderId(orderId)).thenReturn(Optional.of(mockOrder));
+        when(orderRepository.findByOrderId(orderId)).thenReturn(Optional.of(existingOrder));
+        doNothing().when(orderRepository).delete(existingOrder);
 
-        // When
+        // when
         orderService.deleteOrder(orderId);
 
-        // Then
-        verify(orderRepository, times(1)).delete(mockOrder);
-        verify(localCache, times(1)).invalidate(cacheKey);
-        verify(redisTemplate, times(1)).delete(cacheKey);
-        verify(cachePublisher, times(1)).publish(eq("cache-sync"), contains("Deleted order-order:"));
+        // then
+        verify(orderRepository).findByOrderId(orderId);
+        verify(orderRepository).delete(existingOrder);
+    }
+
+    @Test
+    void deleteOrder_NoOrder() {
+        // given
+        String orderId = "NON_EXISTENT";
+
+        when(orderRepository.findByOrderId(orderId)).thenReturn(Optional.empty());
+
+        // when & then
+        RuntimeException exception = assertThrows(RuntimeException.class, () ->
```

Comment:
```suggestion
        RuntimeException exception = assertThrows(OrderNotFoundException.class, () ->
```
`RuntimeException` 이 아닌 새로운 에러를 정의해서 관리하면 좋을것 같아요